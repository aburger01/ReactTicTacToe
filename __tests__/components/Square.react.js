import React from 'react';
import renderer from 'react-test-renderer';
import Square from '../../src/components/Square.react';

test('Does the Square component render correctly?', () => {
  const tree = renderer.create(
    <Square value="O" />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
