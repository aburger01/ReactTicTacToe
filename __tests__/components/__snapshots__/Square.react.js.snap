// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does the Square component render correctly? 1`] = `
<button
  className="square"
  onClick={undefined}
>
  X
</button>
`;

exports[`Does the Squares class name change if winner is true? 1`] = `
<button
  className="square winner"
  onClick={undefined}
/>
`;
